src/Processors/BacktraceLocation.php

```diff
@@ -12,11 +12,12 @@ class BacktraceLocation
      *
      * @var string
      */
-    protected $loggerClass;
+    protected $classesAndFunctionsToIgnore;
 
-    public function __construct($loggerClass = Logger::class)
+    public function __construct(array $classesAndFunctionsToIgnore = [Logger::class])
     {
-        $this->loggerClass = $loggerClass;
+        $this->classesAndFunctionsToIgnore = $classesAndFunctionsToIgnore;
+        $this->classesAndFunctionsToIgnore[] = static::class;
```

Comment:
I would add `BacktraceLocation::class` to the array that gets passed in via the service provider.